praw/models/reddit/more.py

```diff
@@ -72,18 +72,25 @@ def _load_comment(self, comment_id):
     def comments(self, update: bool = True) -> List[Comment]:
         """Fetch and return the comments for a single MoreComments object."""
         if self._comments is None:
+            self._comments = []
             if self.count == 0:  # Handle 'continue this thread'
                 return self._continue_comments(update)
             assert self.children, "Please file a bug report with PRAW."
-            data = {
-                "children": ",".join(self.children),
-                "link_id": self.submission.fullname,
-                "sort": self.submission.comment_sort,
-            }
-            self._comments = self._reddit.post(
-                API_PATH["morechildren"], data=data
-            )
-            if update:
-                for comment in self._comments:
-                    comment.submission = self.submission
+            queue = (
+                self.children.copy()
+            )  # Issue 1401 requires the use of a queue
+            while queue:
+                print("passed")
+                data = {
+                    "children": ",".join(queue[:500]),
+                    "link_id": self.submission.fullname,
+                    "sort": self.submission.comment_sort,
+                }
+                queue = queue[500:]
+                self._comments.extend(
+                    self._reddit.post(API_PATH["morechildren"], data=data)
```

Comment:
[Reddit API documentation](https://www.reddit.com/dev/api#GET_api_morechildren) says the maximum number of items you may request is 100.

Reply:
It takes up to 500 items though.

Reply:
Why do you say that? If the documentation says that 100 is the max, isn't 100 the max?

Reply:
I ran tests with 500 comments, and it did not throw any errors. Furthermore, the error wasn't discovered until someone attempted to do the operation on a post with a lot of top-level comments, so it seems that the limit is higher than 100.